pkg/keyspace/tso_keyspace_group.go

```diff
@@ -143,6 +152,30 @@ func (m *GroupManager) Close() {
 	m.wg.Wait()
 }
 
+func (m *GroupManager) allocNodesForDefaultKeyspaceGroup() {
+	defer logutil.LogPanic()
+	defer m.wg.Done()
+	ticker := time.NewTicker(allocNodesForDefaultKeyspaceGroupInterval)
+	defer ticker.Stop()
+	for {
+		select {
+		case <-m.ctx.Done():
+			return
+		case <-ticker.C:
+		}
+		kg, err := m.GetKeyspaceGroupByID(utils.DefaultKeyspaceGroupID)
+		replica := m.GetNodesNum()
```

Comment:
GetNodesNum() returns the count of registered tso servers, correct? If yes, shall we rename replica to countOfTSONodes or numOfTSONodes or desiredReplicaCount or similar?